keyboards/bemeier/bmek/rules.mk

```diff
@@ -0,0 +1,21 @@
+# MCU name
```

Comment:
I just realized this implementation fails CI (though it works).

Delete this file, and put all its contents into each of `keyboards/bemeier/bmek/rev1/rules.mk`, `keyboards/bemeier/bmek/rev2/rules.mk` and `keyboards/bemeier/bmek/rev3/rules.mk`.

Reply:
the redundancy is mildly infuriating but fine with me :) see cd38d30